deploy/dev-node/nginx/nginx.conf

```diff
@@ -28,6 +28,12 @@ http {
 		    proxy_set_header Host $host;
 		    proxy_set_header X-Real-IP $remote_addr;
 		    proxy_pass http://127.0.0.1:3000;
+
+			proxy_set_header Connection '';
+			proxy_http_version 1.1;
```

Comment:
was the different indentation made intentionally?

Reply:
this is somehow the github diff. locally it looks fine

Reply:
it was a spaces and tabs issue. should be fixed now